### DISCUSSION TITLE
Syntax Highlighting for .log files

### DISCUSSION THREAD

**Discussion by alexxGmZ** [nvim-treesitter/nvim-treesitter#3561] [Ideas]
Hate to compare but polyglot supports syntax highlighting for .log files. Kinda sad that I don't see it in treesitter. Now browsing .log files is not so tasteful in treesitter

**Comment by clason** [comment]
Well, someone (you?) Just needs to write a grammar and make a PR to add it. Parsers are community contributed.

**Reply by theHamsta** [reply]
There is a pinned issue where you can post a language request. I regularly check it to see where language support can be extended
